code/modules/antagonists/heretic/magic/rust_charge.dm

```diff
@@ -0,0 +1,49 @@
+// Rust charge, added so i could put the lion hunter rifle earlier up the tree
```

Comment:
```suggestion
// Rust charge, a charge action that can only be started on rust (and only destroys rust tiles)
```
This comment is redundant, it should explain the present not the past

Reply:
noted